dev-cpp/gtest/files/gtest-1.8.1-no-Werror.patch

```diff
@@ -0,0 +1,14 @@
+Remove "-Werror" as it is not supported in Gentoo and leads to
+unnecessary bugs. e.g., https://github.com/google/googletest/issues/1584
+
+--- a/googletest/cmake/internal_utils.cmake
++++ b/googletest/cmake/internal_utils.cmake
+@@ -99,7 +99,7 @@
+     set(cxx_no_exception_flags "-EHs-c- -D_HAS_EXCEPTIONS=0")
+     set(cxx_no_rtti_flags "-GR-")
+   elseif (CMAKE_COMPILER_IS_GNUCXX)
+-    set(cxx_base_flags "-Wall -Wshadow -Werror")
```

Comment:
I presume upstream isn't going to ever accept it, so I think `sed` would be easier to maintain going forward here.

Reply:
OK.